benchmarks/benchmarks/sparse_csgraph_dijkstra.py

```diff
@@ -40,3 +40,24 @@ def time_dijkstra_multi(self, n, min_only, format):
                  directed=False,
                  indices=self.indices,
                  min_only=min_only)
+
+
+class DijkstraDensity(Benchmark):
+    """
+    Benchmark performance of Dijkstra, adapted from [^1]
+
+    [^1]: https://github.com/scipy/scipy/pull/20717#issuecomment-2562795171
+    """
+    params = [
+        [10, 100, 1000],
+        [0.1, 0.3, 0.5, 0.9],
+    ]
+    param_names = ["n", "density"]
+
+    def setup(self, n, density):
+       G = scipy.sparse.rand(n, n, density, format='csr') * 100
```

Comment:
Nit, we should use the new `scipy.sparse.random_array` interface

Reply:
Good pickup, I'll edit that today.

I was following the example from the issue and didn't think.